travis/publish.sh

```diff
@@ -35,12 +35,12 @@ is_pull_request() {
   fi
 }
 
-is_travis_branch_master() {
-  if [ "${TRAVIS_BRANCH}" = master ]; then
-    echo "[Publishing] Travis branch is master"
+is_travis_branch_master_or_release() {
+  if [[ "${TRAVIS_BRANCH}" == "master" || "${TRAVIS_BRANCH}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    echo "[Publishing] Travis branch is ${TRAVIS_BRANCH}"
     return 0
   else
-    echo "[Not Publishing] Travis branch is not master"
+    echo "[Not Publishing] Travis branch is not master or 0.0.0"
```

Comment:
nit: change 0.0.0 to _a release branch_  ?

Reply:
I want to make it explicit how a release branch should look like.

Reply:
So a release branch should be 0.0.0 ? 😉

Reply:
`s/or 0.0.0/a release branch #.#.#/`